website/static/js/site.js

```diff
@@ -166,7 +166,11 @@
      */
     $.osf.urlParams = function(str) {
         return (str || document.location.search).replace(/(^\?)/,'').split('&')
-            .map(function(n){return n = n.split('='),this[n[0]] = decodeURIComponent(n[1]).replace(/\+/g, ' '),this;}.bind({}))[0];
+            .map(function(n){return n = n.split('='),this[n[0]] = decodeURIComponent(n[1].replace(/\+/g, ' ')),this;}.bind({}))[0];
```

Comment:
After my most recent update, this change was lost - the code was moved to another file, and was overlooked in the merge. I've added it as a separate commit (@4686e85).